docs/convenience_commands.md

```diff
@@ -0,0 +1,126 @@
+# Convenience commands
+
+In addition to the primary commands for the build, imputation, and
+resequencing pipelines, PHGv2 also provides a suite of "convenience
+commands" for miscellaneous "quality of life (QoL)" improvements. In 
+this document, we will discuss the currently available external
+commands for performing highly used tasks.
+
+
+## Convert gVCF files to hVCF files
+
+> Create hVCF files from existing gVCF files created by the PHG
+
+**Command** - `gvcf2hvcf`
+
+**Example**
+
+``` shell
+phg gvcf2hvcf \
+    --bed my/bed/file.bed \
+    --reference-file my/updated/ref/fasta.fa \
+    --gvcf-dir gvcf/directory \
+    --db-path my/phg/db
+```
+
+**Parameters**
+
+| Parameter name     | Description                                                                                                         | Default value         | Required?        |
+|--------------------|---------------------------------------------------------------------------------------------------------------------|-----------------------|------------------|
+| `--bed`            | BED file with entries that define the haplotype boundaries.                                                         | `""`                  | :material-check: |
+| `--gvcf-dir`       | Directory containing bgzipped and CSI indexed gVCF files.                                                           | `""`                  | :material-check: |
+| `--reference-file` | Path to local Reference FASTA file.                                                                                 | `""`                  | :material-check: |
+| `--db-path`        | Folder name where TileDB datasets and AGC record is stored. If not provided, the current working directory is used. | _Current working dir_ |                  |
+
+
+## Convert hVCF files to gVCF files
+
+> Create gVCF files from existing hVCF files created by the PHG
+
+**Command** - `hvcf2gvcf`
+
+**Example**
+
+``` shell
+phg hvcf2gvcf \
+    --reference-file my/updated/ref/fasta.fa \
+    --hvcf-dir hvcf/directory \
+    --db-path my/phg/db
+    --output-dir output/directory/for/gvcfs
+```
+
+**Parameters**
```

Comment:
Add optional parameter "conda-env-prefix".  This currently exists, it just isn't shown in the help message (are optional parameters never shown? what controls that?)

Reply:
I am confused. Downloading the latest version as of this date (`2.3.11.148 `), I am able to see this parameter for `hvcf2gvcf` using the `--help` parameter. Does this not show up for you? Example console output:

<img width="676" alt="Screenshot 2024-07-26 at 4 07 21 PM" src="https://github.com/user-attachments/assets/3bcf371a-c722-4556-bfca-9b0a35e20d53">

Reply:
@btmonier It shows up if you use --help.  It doesn't show up if you just type "./phg hvcf2gvcf".  The latter gives a usage command and only shows the required parameters as missing.  I guess that is correct.  I need to be in the habit of typing "help", but I fall back into what I did for TASSEL, which was just type the command.